### ISSUE TITLE
Package a graph generator library

### ISSUE THREAD

**Issue by RagnarGrootKoerkamp** [RagnarGrootKoerkamp/BAPCtools#124]
Could either be a standalone python script, or part of validation.h

**Comment by mpsijm** [comment]
I don't think that will work... Note that BAPCtools is run using CPython (`python`), whereas the generators are typically run using PyPy.
Problem authors can install `networkx` in their PyPy environment if they so desire, but that's less portable.
One solution we have for visualizers (which typically rely on `matplotlib`), is that they are explicitly run using CPython (either with the `*.cpy` extension and a `languages.yaml` that recognizes this, or with a custom `run` script).
